### ISSUE TITLE
Flickering transparent textures / weird teleporters

### ISSUE THREAD

**Issue by ileakim** [skullernet/q2pro#190]
Transparent textures/brushes flicker and "shift".  This happens almost on every map with transparent brushes. 
https://streamable.com/p0zlw
https://i.imgur.com/xIMoaqW.jpg

Another interesting clip. https://streamable.com/72j78

I've been having these issues on my new pc for a while, everything worked fine before upgrade. I've  tested these same maps with different client (r1q2) and everything works normally. 

RTX 2080 and NVIDIA driver 441.20. Default q2pro graphics settings.

**Comment by perfectcolors** [comment]
`gl_shaders 0` fixed this for me on a RTX 2070 Super.

**Comment by dimmskii** [comment]
Confirming issue. Windows 8.1, nVidia GTX 1060 6GB.

`gl_shaders 0` also seems to fix it.

**Comment by ileakim** [comment]
gl_shaders 0 fixed it, thank you!

**Comment by skullernet** [comment]
Unable to reproduce this in Windows 10 VM with passed through GPU so far. Never seen anything like this in bare metal Linux installation with proprietary NVIDIA driver either.

**Comment by dimmskii** [comment]
It does not happen to me on the Linux x64 builds I make myself either (same PC as the Windows, by the way).

I've found out more:
 - It seems to only happen on Windows 32bit builds.
 - It's something to do with q2pro.exe ; not the game DLL
 - It only affects the pre-built r1826 and r1844  off of the website. r1504 is in the clear.

I was going to test more of the builds to narrow it down, but I only tried building on Linux so far. Are there more archived builds somewhere apart from just those three?

**Comment by skullernet** [comment]
> It seems to only happen on Windows 32bit builds.

So you have actually tested 64-bit builds and they don't have this issue?

> It's something to do with q2pro.exe ; not the game DLL

Well of course, because q2pro.exe contains the renderer.

> It only affects the pre-built r1826 and r1844 off of the website. r1504 is in the clear.

r1504 didn't implement GLSL rendering backend which is affected. It only appeared in r1636 (and became the default it r1645).

> Are there more archived builds somewhere apart from just those three?

Unfortunately, I don't think so. I've attached two builds that may be of interest, can you test them?

[q2pro-r1645.exe.gz](https://github.com/skullernet/q2pro/files/4232565/q2pro-r1645.exe.gz)
[q2pro-r1703.exe.gz](https://github.com/skullernet/q2pro/files/4232566/q2pro-r1703.exe.gz)

**Comment by dimmskii** [comment]
@skullernet
So I tested both of those builds. Both of them indeed have this same issue.

**Comment by dimmskii** [comment]
It behaves weird as well. I had another user with limited privileges and UAC temporarily enabled a few days ago. It didn't happen under that user. Upon switching back and forth between the accounts, ASUS GPU Tweak kept popping up, so I closed it (forgetting that it completely closes down the program).

I though it was the GPU Tweak speeding up the VRAM too much (can have similar results), but no. I couldn't get the bug reproduced for a while. Interestingly enough, it came back as soon as UAC and admin approval mode was fully disabled. After that, I went back to the non-admin account and the issue didn't occur. Back to the admin: the issue is present again!

So... this only happens with elevated privileges? I refuse to believe this. If this were somehow the case, I'd start pointing fingers at the nVidia drivers!
